website/docs/components/rich-tooltip/partials/code/how-to-use.md

```diff
@@ -0,0 +1,285 @@
+  </RT.Bubble>
+</Hds::RichTooltip>
+```
+
+### Collision detection
+
+The RichTooltip component automatically adapts its alignment depending on its position relative to the viewport to avoid "collisions" with the browser window boundaries. This means that when an end-user scrolls the page, or resizes the browser, the position of the tooltip on the page dynamically adapts to these changes (along the two axes, main and secondary).
+
+It's possible to disable this behavior setting the `@enableCollisionDetection` argument to `false`.
+
+It's also possible to customize this behaviour using the values:
+- `flip` - adapts the position only along its [axis of alignment](https://floating-ui.com/docs/flip#mainaxis)
+- `shift` - adapts the position only along its [side axis](https://floating-ui.com/docs/shift#mainaxis)
+- `auto` - uses the side where there is [more space available](https://floating-ui.com/docs/autoPlacement) (in relation to the viewport)
+
+The default option is a combination of `flip`+`shift` and is the suggested one, but there may be use cases where the other options work better in a specific context/use case.
+
+!!! Info
+
+More in-depth explanations about the different alignment algorithms and how they work can be found in the Floating UI documentation. See [flip](https://floating-ui.com/docs/flip), [shift](https://floating-ui.com/docs/shift), and [autoPlacement](https://floating-ui.com/docs/autoPlacement)).
+
+!!!
+
+In the example below we use an `auto` placement: try to scroll and/or resize the page and see how the tooltip changes its position automatically:
+
+```handlebars
+<Hds::RichTooltip @enableClickEvents={{true}} as |RT|>
+  <RT.Toggle @size="medium" @text="More info" @icon="info" />
+  <RT.Bubble @enableCollisionDetection="auto">
+    <Hds::Text::Display @tag="h4" @size="200">Some title</Hds::Text::Display>
+    <Hds::Text::Body @tag="p" @size="200">Some descriptive information that spans multiple lines</Hds::Text::Body>
+  </RT.Bubble>
+</Hds::RichTooltip>
+```
+
+### Fixed width/height
+
+By default the size of the tooltip automatically adapts to the size of its content (with a max-width of `280px`). It's possible to assign fixed `width` and/or `height` to the tooltip providing the `@width`/`@height` arguments:
+
+```handlebars
+<Hds::RichTooltip as |RT|>
+  <RT.Toggle @size="medium" @text="More info" @icon="info" />
+  <RT.Bubble @width="450px" @height="200px">
+    <Hds::Text::Display @tag="h4" @size="200">Some title</Hds::Text::Display>
+    <Hds::Text::Body @tag="p" @size="200">Some descriptive information</Hds::Text::Body>
+  </RT.Bubble>
+</Hds::RichTooltip>
+```
+
+### Generic toggle
+
+There may be special use cases in which the standard text/icon-based toggle doesn't work in a specific context or design. For this reason custom content can be `yield` to the `Toggle` element, but this should be considered an option of last resort, because it could result in a non-accessible implementation.
```

Comment:
```suggestion
There may be special use cases in which the standard text or icon-based toggle doesn't work in a specific context or design. For this reason, custom content can be yielded to the `Toggle` element, but this should be considered an option of last resort, because it could result in a non-accessible implementation.
```